app/assets/src/components/utils/format.js

```diff
@@ -18,3 +18,11 @@ export const formatWithUnits = (number, unitFactor, units) => {
 
   return `${unitIndex > 0 ? number.toFixed(1) : number} ${units[unitIndex]}`;
 };
+
+export const formatSentenceCase = string => {
```

Comment:
There's already a function in `StringHelper` called `capitalizeFirstLetter`. I would use that instead.

As a side note, I typically avoid using regex when simpler tools will work, because regex is less human-readable and it makes the whole function harder to understand. Would it be burdensome for you to use string functions instead of regex, where applicable?